src/components/user-menu/user-menu.tsx

```diff
@@ -0,0 +1,60 @@
+import { NavLink } from '@remix-run/react';
+import classNames from 'classnames';
+import { useUserInfo } from '~/lib/hooks';
+import { Avatar } from '../avatar/avatar';
+import {
+    DropdownMenu,
+    DropdownMenuItem,
+    DropdownMenuSeparator,
+} from '../dropdown-menu/dropdown-menu';
+import { ChevronDownIcon } from '../icons';
+
+import styles from './user-menu.module.scss';
+
+export const UserMenu = () => {
+    const { isLoggedIn, user } = useUserInfo();
+
+    if (!isLoggedIn) {
+        return (
+            <NavLink className={classNames(styles.link, styles.root)} to={'/login'}>
+                <Avatar imageSrc={user?.profile?.photo?.url} />
+                Log In
+            </NavLink>
+        );
+    }
+
+    return (
+        <DropdownMenu
+            trigger={
+                <button className={styles.root}>
+                    <Avatar imageSrc={user?.profile?.photo?.url} />
+                    <ChevronDownIcon width={10} height={10} />
```

Comment:
we have `DropdownIcon`, can't it be used here?

Reply:
you are right, removed that new icon and used `DropdownIcon` here